crypto/account.go

```diff
@@ -198,6 +200,20 @@ func MakeLogicSigAccountEscrow(program []byte, args [][]byte) LogicSigAccount {
 	}
 }
 
+// MakeLogicSigAccountEscrowChecked creates a new escrow LogicSigAccount.
+// The address of this account will be a hash of its program.
+func MakeLogicSigAccountEscrowChecked(program []byte, args [][]byte) (LogicSigAccount, error) {
+	if err := sanityCheckProgram(program); err != nil {
```

Comment:
nit: it would be a little nicer to call `MakeLogicSig` here to unify all the code paths.

Reply:
yes, that makes sense